src/plot.js

```diff
@@ -100,6 +100,7 @@ export function plot(options = {}) {
     for (const mark of marks) {
       if (mark.facet === "exclude") {
         const facetState = facetStateByMark.get(mark);
+        if (facetState === undefined) throw new Error(`the facet: "exclude" option is restricted to top-level facets`);
```

Comment:
This isn’t accurate—you can also use `facet: "exclude"` if you specify mark-level faceting, e.g.

```js
Plot.dot(penguins, {facet: "exclude", fx: "sex", fy: "species", x: "culmen_depth_mm", y: "culmen_length_mm", r: 2, fill: "#ddd"}),
```